app/controllers/alerts_list_controller.rb

```diff
@@ -16,6 +17,15 @@ def index
     redirect_to :action => 'show'
   end
 
+  def class_icons
+    res = {}
+    icons.each do |k, _val|
+      file_name = "100/#{k.to_s.underscore}.png"
```

Comment:
It is available now :) (You need current master for both manageiq and ui-classic)

Then to use it, all you have to do is add a static method to the relevant decorator...

So you'd go to `MiqPolicyDecorator` for example, add something like `def self.listicon_image; "100/miq_policy.png"; end` .. aaand then `MiqPolicy.decorate.listicon_image` will work.